securing_apps/topics/oidc/java/jboss-adapter.adoc

```diff
@@ -166,16 +165,19 @@ endif::[]
 {appserver_name} has built-in support for single sign-on for web applications deployed to the same {appserver_name}
 instance. This should not be enabled when using {project_name}.
 
-===== Required Per WAR Configuration
+===== Configuring a WAR as required
```

Comment:
This title is confusing before and still now? Looks like this is about "Protecting/Securing a WAR" ?

The same comment applies to other adapters.

Reply:
Thanks, @pedroigor. I am making it "Securing a WAR"
I am making a number of other changes and will value your feedback.